var/spack/repos/builtin/packages/podio/package.py

```diff
@@ -72,12 +73,22 @@ class Podio(CMakePackage):
 
     depends_on("cxx", type="build")  # generated
 
+    _cxxstd_values = (conditional("17", when="@:1.2"), conditional("20", when="@0.15:"))
```

Comment:
```suggestion
    _cxxstd_values = (conditional("17", when="@:1.2"), conditional("20", when="@0.14.1:"))
```
Just because I check when we actually introduced c++20 as a valid option. It was [this commit](https://github.com/AIDASoft/podio/commit/9994850569be09453ef6e24acb8613abb0c8cc5a). So technically with the versions that are still available from spack, `20` could be made unconditional.